sycl/test-e2e/AddressSanitizer/test.cpp

```diff
@@ -0,0 +1,41 @@
+// REQUIRES: linux
+// UNSUPPORTED: gpu, cuda, hip
```

Comment:
Alternatively, this should work too:
```suggestion
// REQUIRES: linux, cpu
```

Reply:
Thanks!